incubator/binding-mqtt.spec/src/main/resources/META-INF/zilla/mqtt.idl

```diff
@@ -17,21 +17,36 @@ scope mqtt
 {
     enum MqttSubscribeFlags (uint8)
     {
-        SEND_RETAINED(0),
-        RETAIN_AS_PUBLISHED(1),
-        NO_LOCAL(2)
+        QOS0(0),
+        QOS1(1),
+        QOS2(2),
+        SEND_RETAINED(3),
+        RETAIN_AS_PUBLISHED(4),
+        NO_LOCAL(5),
+        RETAIN(6)
```

Comment:
I implemented it this way, since it's better aligned with the protocol (QOS data is inside the "flags"). But since this the internal representation, we can do whatever we like. I'll make the change